packages/@okta/vuepress-site/docs/reference/api/myaccount-migration/index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: MyAccount migration
+category: management
+---
+
+# MyAccount API migration guide
+
+<ApiLifecycle access="ie" />
+
+This guide describes how to migrate from the `v1` version to the `idp` version of the MyAccount API. See [MyAccount API](/docs/reference/api/myaccount/) for the reference documentation of the latest version.
+
+## OIE migration
+
+The `idp` version of the MyAccount API only works on Okta Identity Engine. See [Identity Engine upgrade overview](/docs/guides/oie-upgrade-overview/main/) if you need to upgrade.
+
+## Use the Postman collection
+
+To run the Postman collection of the `idp` version of the API, you need an end-user access token. Use an [SDK](/docs/guides/auth-js/-/main/) to get the token.
+
+[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/9cb68745dbf85ae3a871)
+
+## Use the Auth.js SDK
+
+To help you with your migration, for a limited time you can call endpoints of both versions of the API.
```

Comment:
We no longer allow customers to enable using the old API, maybe worth changing this line to indicate that is only available if the customer is already using the deprecated api.